pkg/application/client.go

```diff
@@ -688,9 +688,34 @@ func (c *ApplicationClient) getApplicationDetails(ctx context.Context, appID str
 		return nil, fmt.Errorf("failed to get OAuth details: %w", err)
 	}
 
+	authorizedAPIs, err := c.GetAuthorizedAPIs(ctx, appID)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get authorized APIs: %w", err)
+	}
+
+	authorizedScopes := ""
+	scopeSet := make(map[string]struct{})
+	if authorizedAPIs != nil {
+		for _, api := range *authorizedAPIs {
+			if api.AuthorizedScopes != nil {
+				for _, scope := range *api.AuthorizedScopes {
+					scopeSet[*scope.Name] = struct{}{}
+				}
+			}
+		}
+	}
+
+	var scopes []string
+	for scope := range scopeSet {
+		scopes = append(scopes, scope)
+	}
+	// todo: add open id connect scopes based on claim configuration
+	authorizedScopes = strings.Join(scopes, " ") + " openid profile email" 
```

Comment:
Shall we avoid returning the OpenID scopes for now? If the app doesn’t have them (e.g., removed after creation), we might be returning incorrect info. Let’s only include them once the TODO is properly

Reply:
These are default stuff. It's fine